schunk_gripper_driver/schunk_gripper_driver/driver.py

```diff
@@ -891,26 +902,27 @@ def _fast_stop_cb(
         response.message = gripper["driver"].get_status_diagnostics()
         return response
 
-    def _move_to_absolute_position_cb(
+    def _move_to_position_cb(
         self,
-        request: MoveToAbsolutePosition.Request,
-        response: MoveToAbsolutePosition.Response,
+        request: Any,
+        response: Any,
         gripper: Gripper,
+        service_name: str,
```

Comment:
I suggest using the `is_absolute: bool` flag here in combination with the `if "absolute" in ..` from below. The service name here seems to be a piece of information that the `_move_to_position_cb` shouldn't need to know nor parse.